### PR TITLE
Update for conduit 1.0

### PR DESCRIPTION
**Pull request by joeyadams**
This switches to conduit 1.0, dropping support for conduit 0.5 and bumping the major version number of this package.

I'm not a heavy conduit user (yet), so please look over these changes.

### CODE REVIEW

File: cereal-conduit.cabal [hunk 1]
```diff
@@ -25,15 +30,15 @@ library
...
-                 , cereal >= 0.3.1.0
+    build-depends: base
+                 , conduit
+                 , cereal
                  , bytestring
                  --, test-framework-hunit
                  , HUnit
                  , resourcet
                  , mtl
-                 , transformers >= 0.2.0.0
```
Comment (litherum) [inline cereal-conduit.cabal:36]: Why removing these requirements?

**Comment by litherum** [comment]
Looks pretty good, only one minor question. Thanks for doing this for me!!!

File: cereal-conduit.cabal (hunk 1 above)
Comment (joeyadams) [inline cereal-conduit.cabal:36]: See the commit message for 0a0e6ec.  Bounds in the test-suite section are unnecessary, unless I'm mistaken.  This keeps us from having to maintain the bounds in two sections.

**Comment by litherum** [comment]
http://hackage.haskell.org/package/cereal-conduit

**Pull request merged**